benchmark/scenarios/test/test_write_request_load.py

```diff
@@ -167,6 +168,39 @@ def assert_created(returned_datasets):
         d2.addCallback(assert_created)
         s.stop()
 
+    @capture_logging(None)
+    def test_cleanup_deletes_dataset(self, _logger):
+        """
+        ``write_request_load_scenario`` deletes the dataset created by
+        the setup when the scenario is stopped.
+        """
+        c = Clock()
+        cluster = self.make_cluster(self.get_fake_flocker_client_instance())
```

Comment:
Extract the control service from the cluster here, to avoid using `s.scenario_setup.control_service` later.  You should not be relying on the fact that the scenario setup object also extracts the control service.

:question: This should be applied to all the tests in this file, and `WriteRequest` should be modified to accept the control service instead of the cluster (since it doesn't need the cluster object).  However, this changes a lot of lines that are not part of this PR.